packages/utils/src/featureFlags.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2022 Beyond Essential Systems Pty Ltd
+ */
+
+import { requireEnv } from './requireEnv';
+
+/**
+ * Simple feature flag functionality implemented via env vars
+ *
+ * @param {string} featureName
+ * @returns {boolean}
+ */
+export const isFeatureEnabled = featureName => {
+  return requireEnv(`FEATURE_FLAG_${featureName}`).toLowerCase() === 'on';
+};
```

Comment:
Decided to go with something super simple here. I actually toyed up going with a 3rd party package or something more complex, but couldn't really justify it.

Some feature that could be super useful to have:
1. Different toggle settings for prod/dev/local envs
2. Ability to switch a toggle in a live environment
3. Ability to switch a toggle for certain users/permissions

But for now just managing it via `.env` variables will be simple and fit easily with our CI/CD system 👍

Reply:
Really really like the simplicity of the solution - thanks for not being sucked in by a flashy third party solution when really it's a straightforward and isolated thing atm!